apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/UrlSignerSnippets.cs

```diff
@@ -64,6 +64,41 @@ public async Task SignedURLGet()
             Assert.Equal(_fixture.HelloWorldContent, content);
         }
 
+        [Fact]
+        public async Task HmacSignedURLGet()
+        {
+            var bucketName = _fixture.BucketName;
+            var objectName = _fixture.HelloStorageObjectName;
+            var credential = (await GoogleCredential.GetApplicationDefaultAsync()).UnderlyingCredential as ServiceAccountCredential;
```

Comment:
I'd suggest using a cast here instead of `as` so that we get a useful exception here instead of a NullReferenceException later on, if the credential isn't a service account.

Reply:
Changing here and from where I copied it from :)